mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketClient.java

```diff
@@ -35,6 +40,15 @@ public MqttWebSocketClient(URI serverUri, MqttMessageListener listener) {
         this.listener = listener;
     }
 
+    @Override
+    public boolean connectBlocking(final long timeout, final TimeUnit timeUnit) throws InterruptedException {
+        final boolean connected = super.connectBlocking(timeout, timeUnit);
```

Comment:
IMO this would be cleaner

if (!super.connectBlocking(timeout, timeUnit)) {
            throw new RuntimeException(isNotBlank(connectionError) ? connectionError : "Cannot connect: unknown error");
        }
return true;

Reply:
Done